### DISCUSSION TITLE
Is it possible to run lua?

### DISCUSSION THREAD

**Discussion by joshpetit** [dcampos/nvim-snippy#32] [Q&A] [answered]
Hey! Thanks a lot for this plugin it's really awesome.

I was wondering if it's possible to evaluate lua in the snippet files? I see the example with the `strftime` function call, but would it be possible to do something like `lua print('')`? If so what would the syntax look like for something like that?

Thank you!

**Comment by dcampos** [comment, marked as answer]
Hi, I'm glad you liked the plugin!

In the current version, you can evaluate Lua by starting the eval block with `!lua`:

```vim-snippet
snippet luatime
  Current time is: `!lua os.date("%T")`
```
---
_Older answer (still possible):_

It is possible to evaluate Lua code by using the luaeval function. For example:

```vim-snippet
snippet luatime
  Current time is: `luaeval('os.date("%T")')`
```
